Makefile

```diff
@@ -61,6 +61,10 @@ test: manifests generate fmt vet ## Run tests.
 	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.2/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
 
+e2e-tests:
+	k3d cluster delete e2e-test-cluster
+	k3d cluster create e2e-test-cluster
```

Comment:
I guess we will run the tests locally too at one point. This is what we're aiming for, isn't it?

Reply:
Yes. I decided to use the k3d to allow us to run the tests locally. But I'm not working helper script to allow that yet.